addons/docs/src/frameworks/react/jsxDecorator.tsx

```diff
@@ -84,14 +84,20 @@ export const renderJsx = (code: React.ReactElement, options: JSXOptions) => {
     }
   }
 
-  const opts =
+  const displayNameDefaults =
     typeof options.displayName === 'string'
-      ? {
-          ...options,
-          showFunctions: true,
-          displayName: () => options.displayName,
-        }
-      : options;
+      ? { showFunctions: true, displayName: () => options.displayName }
+      : {};
+
+  const filterDefaults = {
+    filterProps: (value: any, key: string): boolean => value != null,
```

Comment:
Why filter `null`? Why not just `undefined`? @phated

Reply:
@shilman no particular reason. I've just pushed another commit that allows null values to be displayed by default.